src/core/Modal/Modal.tsx

```diff
@@ -0,0 +1,51 @@
+import { Modal as MuiModal, Button } from '@material-ui/core';
+import React from 'react';
+import useStyles from './styles';
+
+interface ModalProps {
+  isOpen: boolean;
+  handleClose: () => void;
+  hasCloseBtn: boolean;
+}
+
+const Modal: React.FC<ModalProps> = ({
+  children,
+  hasCloseBtn,
+  isOpen,
+  handleClose,
+}) => {
+  const classes = useStyles();
+  const body = (
+    <div className={classes.root}>
+      {hasCloseBtn && (
+        <span className={classes.modalContainerClose}>
+          <Button
+            variant="outlined"
+            className={classes.closeButton}
+            onClick={handleClose}
+          >
+            &#x2715;
+          </Button>
+        </span>
```

Comment:
Can we just move the button to the top right instead of using a span, just using `closeButton` css